### PR TITLE
avoid error due to finite precision  error in M2 pon creation

### PR DESCRIPTION
**Pull request by davidbenjamin**
@LeeTL1220 for the REBC pon.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java [hunk 1]
```diff
@@ -181,14 +183,22 @@ private static final int altCount(final Genotype g) {
...
         final double hetPrior = alleleFrequency * (1 - alleleFrequency) * 2;
         final double homPrior = MathUtils.square(alleleFrequency);
         final double hetLikelihood = MathUtils.binomialProbability(totalCount, altCount, 0.5);
         final double homLikelihood = MathUtils.binomialProbability(totalCount, altCount, 0.98);
 
         final double artifactLikelihood = new BetaBinomialDistribution(null, ARTIFACT_ALPHA, ARTIFACT_BETA, totalCount).probability(altCount);
 
-        return MathUtils.normalizeFromRealSpace(new double[] {hetPrior * hetLikelihood + homPrior * homLikelihood, ARTIFACT_PRIOR * artifactLikelihood})[0];
+        final double[] relativeProbsOfHetHomArtifact = {hetPrior * hetLikelihood + homPrior * homLikelihood, ARTIFACT_PRIOR * artifactLikelihood};
+        if (MathUtils.sum(relativeProbsOfHetHomArtifact) < 0) {
```
Comment (LeeTL1220) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java]: `return max(..., 0);`

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java [hunk 2]
```diff
@@ -181,14 +183,22 @@ private static final int altCount(final Genotype g) {
...
+            return 0;
+        }
+        return MathUtils.normalizeFromRealSpace(relativeProbsOfHetHomArtifact)[0];
```
Comment (LeeTL1220) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java]: The indices look like they have special meaning.  Can you describe, in a comment, the meaning of the values in [0] and [1]?

Comment (davidbenjamin) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java]: done

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java (hunk 1 above)
Comment (davidbenjamin) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java]: done

**Pull request merged**